packages/pluggableWidgets/rating-web/src/components/Rating.tsx

```diff
@@ -0,0 +1,74 @@
+import { createElement, CSSProperties, ReactElement, useState } from "react";
+import classNames from "classnames";
+
+export interface RatingProps {
+    animated: boolean;
+    className?: string;
+    disabled: boolean;
+    emptyIcon: ReactElement;
+    fullIcon: ReactElement;
+    maximumValue: number;
+    onChange?: (value: number) => void;
+    style?: CSSProperties;
+    tabIndex?: number;
+    value: number;
+}
+
+export function Rating(props: RatingProps): ReactElement {
+    const [hover, setHover] = useState<undefined | number>(undefined);
+    const onClickAction = !props.disabled
+        ? (currentIndex: number): void => {
+              if (props.value === currentIndex) {
+                  props.onChange?.(0);
+              } else {
+                  props.onChange?.(currentIndex);
+              }
+          }
+        : undefined;
+
+    return (
+        <div
+            className={classNames("widget-rating", props.className)}
```

Comment:
IMO a string template is more appropriate here. What do you think?

Reply:
The lib is already imported to be used in line 42, so it will not hurt to use it also here.. as it will just interpolate

Reply:
Also for the sake of consistency just using the library is better